assets/package.json

```diff
@@ -26,43 +26,24 @@
   "homepage": "https://github.com/elixir-lang/ex_doc#readme",
   "browserslist": "last 2 versions",
   "devDependencies": {
-    "@babel/cli": "^7.8.4",
-    "@babel/core": "^7.9.0",
-    "@babel/preset-env": "^7.9.5",
-    "@babel/runtime": "^7.9.2",
-    "autoprefixer": "^9.7.6",
-    "babel-eslint": "^9.0.0",
-    "babel-loader": "^8.1.0",
-    "clean-webpack-plugin": "^3.0.0",
-    "css-loader": "^1.0.1",
-    "eslint": "^5.16.0",
-    "eslint-config-standard": "^12.0.0",
-    "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-node": "^7.0.1",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1",
-    "file-loader": "^2.0.0",
+    "esbuild": "^0.15.7",
+    "eslint": "^8.23.1",
+    "eslint-config-standard": "^17.0.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-n": "^15.2.5",
+    "eslint-plugin-promise": "^6.0.1",
+    "fs-extra": "^10.1.0",
     "handlebars": "^4.7.7",
-    "handlebars-loader": "^1.7.1",
     "karma": "^6.3.16",
-    "karma-chai-plugins": "^0.6.0",
-    "karma-chrome-launcher": "^0.2.0",
-    "karma-firefox-launcher": "^0.1.6",
-    "karma-mocha": "^0.2.0",
+    "karma-chai-plugins": "^0.9.0",
+    "karma-chrome-launcher": "^3.1.1",
+    "karma-esbuild": "^2.2.5",
+    "karma-firefox-launcher": "^2.1.2",
```

Comment:
We may no longer need this, looking at:

https://github.com/elixir-lang/ex_doc/blob/5929958b6f9bf56ea4bb31de647fab294c9f8396/assets/karma.conf.js#L20

I assume `TRAVIS` is not set anywhere :)